Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -368,9 +361,10 @@ final class VisibleItemsProvider {
   // MARK: Private
 
   private let layoutItemTypeEnumerator: LayoutItemTypeEnumerator
-  private let visibleMonthHeaderHeightTracker: VisibleMonthHeaderHeightTracker
   private let frameProvider: FrameProvider
 
+  private var monthHeaderViewsForViewDifferentiators = [_CalendarItemViewDifferentiator: UIView]()
```

Comment:
It's easy enough to just maintain a separate dictionary of sizing views. Then there's no chance we'll accidentally use a real month header view from the reuse manager to do sizing.